test/test_images/latencymako/dev.config

```diff
@@ -33,11 +33,19 @@ metric_info_list: {
   value_key: "pe"
   label: "publish-errors"
 }
+metric_info_list: {
+  value_key: "st"
+  label: "send-throughput"
+}
 metric_info_list: {
   value_key: "dl"
   label: "deliver-latency"
 }
 metric_info_list: {
   value_key: "de"
   label: "deliver-errors"
+}
+metric_info_list: {
+  value_key: "dt"
+  label: "delivery-throughput"
```

Comment:
Use `deliver-throughput` instead of `delivery-throughput` (or change the others to use `delivery`)